testdata/noAdditionalProperties.schema.json

```diff
@@ -0,0 +1,38 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "additionalProperties": false,
+    "properties": {
+        "object": {
+            "additionalProperties": false,
+            "properties": {},
+            "type": "object"
+        },
+        "objectOfObjects": {
+            "additionalProperties": false,
+            "patternProperties": {
+                "^.*$": {
+                    "type": "object"
```

Comment:
`"additionalProperties": false,"` is missing for the inner object. Converting to draft until this is fixed.

Reply:
This is fixed in the latest revision.